packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js

```diff
@@ -201,16 +324,17 @@ function createText(options = {}) {
     value
   } = options;
 
-  return render(
+  return render(WithFormContext(
```

Comment:
Why is this necessary?

Reply:
The FormContext is needed to evaluate FEEL expressions, e.g. it's loading the full form data state into a single component.

In the form field unit tests setup, we mock it to encapsulate the test logic. If we would not mock it, it would simply blow up.